etl/namespaces/rdf.py

```diff
@@ -0,0 +1,7 @@
+from pyoxigraph import NamedNode
+
+
+class RDF:
+    """A class containing RDF Nodes of the default RDF namespace."""
+
+    TYPE = NamedNode("http://www.w3.org/1999/02/22-rdf-syntax-ns#type")
```

Comment:
Nit: don't repeat the base IRI, the pattern you had in ARKG is better.

Reply:
I have used this pattern in the `RDF` and `Schema` classes.